src/utils/hooks/useListApplications.tsx

```diff
@@ -14,7 +15,8 @@ const useListApplications = ({
   type,
   ...queryFilters
 }: BasicStringObject) => {
-  const APPLICATION_FILTERS = useApplicationFilters()
+  const { preferences } = usePrefs()
+  const FILTER_DEFINITIONS = useGetFilterDefinitions(preferences?.defaultListFilters || [])
```

Comment:
As per my above comment, you don't need these pref imports here if you do it all in the hook.